src/routes/blog/post/announcing-db-operators/+page.markdoc

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "Announcing DB operators: Update multiple fields without fetching the entire row"
+description: With Appwrite’s DB operators, make precise, atomic updates across fields in a single call. No redundant reads or race conditions.
+date: 2025-11-04
+cover: /images/blog/announcing-db-operators/cover.png
+timeToRead: 5
+author: jake-barnby
+category: announcement
+featured: false
+---
+If you’ve ever needed to update a single field in a row: increment a counter, add a tag, or adjust an expiry date, you probably had to read the full row, make the change locally, and then write the entire row back.
+
+While that’s fine at a small scale, it becomes a real bottleneck once you have concurrent updates or high-traffic workloads. Each read–modify–write cycle adds latency, increases bandwidth usage, and opens up the risk of **lost updates** when multiple writes overlap.
+
+To make this faster and safer, Appwrite is introducing **DB operators.** A new way to perform **inline, atomic updates** directly at the storage layer. Instead of sending new values, you describe the action: increment, append, replace, or adjust. Appwrite applies that instruction directly in the database, performing the change in one atomic step and keeping your data consistent without extra work in your code.
+
+# Fixing the read–modify–write problem
+
+DB operators eliminate the friction that comes with traditional updates. There’s no need to fetch an entire row just to bump a counter, tweak a tag list, or roll a date forward. You send only the operation, and Appwrite applies it in one atomic step.
+
+This approach removes bulky payloads, avoids race conditions caused by read–modify–write loops, and reduces the number of network round-trips. The result is simpler update logic and more predictable performance, especially under concurrent workloads.
+
+# Currently supported operators
+
+With this release, Appwrite introduces **DB operators v1,** a set of operations that let you perform atomic updates across different field types. These are the **currently supported operator types**, built to cover the most common update scenarios developers handle every day.
+
+- **Numeric operators**
+
+Perform arithmetic operations directly on numeric fields without reading the row first.
+
+`increment`, `decrement`, `multiply`, `divide`, `modulo`
```

Comment:
We also have a `power` operator

Let's compare the list of all the operators with the docs PR once